tips/TIP-0025/core-rest-api.yaml

```diff
@@ -2865,13 +2914,35 @@ components:
                 vByteFactoKey:
                   description: Defines the factor to be used for key/lookup generating fields.
                   type: integer
+            baseToken:
```

Comment:
I think this should be less indented so it's on the same level as `protocol`

Reply:
fixed in https://github.com/iotaledger/tips/pull/57/commits/b6cdd1e3d16ead8fdbd233b64f37aee0cc4bc151